test/libc/calls/cachestat_test.c

```diff
@@ -0,0 +1,119 @@
+/*-*- mode:c;indent-tabs-mode:nil;c-basic-offset:2;tab-width:8;coding:utf-8 -*-│
+│vi: set net ft=c ts=2 sts=2 sw=2 fenc=utf-8                                :vi│
+╞══════════════════════════════════════════════════════════════════════════════╡
+│ Copyright 2022 Justine Alexandra Roberts Tunney                              │
```

Comment:
[optional] This is test code. Feel free to make your mark in this file rather than attributing it to me.